src/game-based-learning.svg

```diff
@@ -0,0 +1 @@
.89,0,0,0,50,17.24h4.42A11.89,11.89,0,0,1,66.32,29.13Z"/><rect class="cls-4" x="1" y="17.06" width="65.14" height="39.23" rx="11.38"/><rect class="cls-5" x="51.03" y="26.27" width="6.59" height="6.59" rx="3"/><path class="cls-6" d="M56.92,28.89a3.38,3.38,0,0,1-3.46,3.29,3.59,3.59,0,0,1-1.73-.45.25.25,0,0,0,.12-.08,2.39,2.39,0,0,0,.66.08,2.84,2.84,0,1,0-.66-5.6l-.12-.08a3.6,3.6,0,0,1,1.73-.46A3.39,3.39,0,0,1,56.92,28.89Z"/><rect class="cls-4" x="51.03" y="26.27" width="6.59" height="6.59" rx="3.29"/><rect class="cls-5" x="51.03" y="41.18" width="6.59" height="6.59" rx="3"/><path class="cls-6" d="M56.92,43.8a3.38,3.38,0,0,1-3.46,3.29,3.48,3.48,0,0,1-1.73-.45.25.25,0,0,0,.12-.08,2.39,2.39,0,0,0,.66.08,2.84,2.84,0,1,0-.66-5.6.35.35,0,0,0-.12-.07,3.5,3.5,0,0,1,1.73-.47A3.39,3.39,0,0,1,56.92,43.8Z"/><rect class="cls-4" x="51.03" y="41.18" width="6.59" height="6.59" rx="3.29"/><rect class="cls-5" x="36.91" y="26.27" width="6.59" height="6.59" rx="3"/><path class="cls-6" d="M42.79,28.89a3.38,3.38,0,0,1-3.46,3.29,3.55,3.55,0,0,1-1.72-.45.32.32,0,0,0,.12-.08,2.39,2.39,0,0,0,.66.08,2.84,2.84,0,1,0-.66-5.6,1.25,1.25,0,0,0-.12-.08,3.56,3.56,0,0,1,1.72-.46A3.39,3.39,0,0,1,42.79,28.89Z"/><rect class="cls-4" x="36.91" y="26.27" width="6.59" height="6.59" rx="3.29"/><rect class="cls-5" x="36.91" y="41.18" width="6.59" height="6.59" rx="3"/><path class="cls-6" d="M42.79,43.8a3.38,3.38,0,0,1-3.46,3.29,3.45,3.45,0,0,1-1.72-.45.32.32,0,0,0,.12-.08,2.39,2.39,0,0,0,.66.08,2.84,2.84,0,1,0-.66-5.6.47.47,0,0,0-.12-.07,3.46,3.46,0,0,1,1.72-.47A3.39,3.39,0,0,1,42.79,43.8Z"/><rect class="cls-4" x="36.91" y="41.18" width="6.59" height="6.59" rx="3.29"/><path class="cls-7" d="M28.45,36.46a3,3,0,0,1-3,3l-5.16.07.07,5.34a3,3,0,0,1-6,0s-.06-2.2-.11-5.27l-5.19.07a3,3,0,0,1,0-6l5.11-.07c0-1.73,0-3.49,0-5.12a3,3,0,0,1,6,0l.06,5.05,5.24-.07a3,3,0,0,1,3,3Z"/><path class="cls-4" d="M28.45,36.46a3,3,0,0,1-3,3l-5.16.07.07,5.34a3,3,0,0,1-6,0s-.06-2.2-.11-5.27l-5.19.07a3,3,0,0,1,0-6l5.11-.07c0-1.73,0-3.49,0-5.12a3,3,0,0,1,6,0l.06,5.05,5.24-.07a3,3,0,0,1,3,3Z"/><path class="cls-8" d="M29.58,17V12a6,6,0,0,1,6-6H44.9a1,1,0,0,0,1-1V1h6V6a6,6,0,0,1-2.64,5c-.12.08-.24.16-.37.23s-.25.14-.39.2a5.3,5.3,0,0,1-1.25.44A5.54,5.54,0,0,1,45.9,12H36.48l-.1,0a.65.65,0,0,0-.36.15.79.79,0,0,0-.15.12,1,1,0,0,0-.29.7v4Z"/><g class="cls-9"><path class="cls-10" d="M32.71,17V13a1,1,0,0,1,1-1h3l-.2,0a.73.73,0,0,0-.36.15l-.15.12a1,1,0,0,0-.29.7v4Z"/></g><g class="cls-9"><path class="cls-10" d="M42.78,12h3a6,6,0,0,0,1.35-.15,5.79,5.79,0,0,0,1.25-.44c.14-.06.26-.13.39-.2s.25-.15.37-.23a6,6,0,0,0,2.64-5V1h-3V6A6,6,0,0,1,42.78,12Z"/></g><path class="cls-4" d="M29.7,17V12a6,6,0,0,1,6-6h9.07a1,1,0,0,0,1-1V1h6V6a6,6,0,0,1-2.64,5c-.12.08-.24.16-.37.23s-.25.14-.39.2a4.89,4.89,0,0,1-1.25.44,5.54,5.54,0,0,1-1.35.16H36.6l-.1,0a.73.73,0,0,0-.36.15.79.79,0,0,0-.15.12,1,1,0,0,0-.29.7v4Z"/><rect class="cls-11" x="30.71" y="14.03" width="4" height="2"/><rect class="cls-11" x="46.78" y="2.01" width="4" height="2"/></g></g></g></svg>
```

Comment:
Can you put this through [SVG OMG](https://jakearchibald.github.io/svgomg/)? It does some optimizations and stuff.

Also, we want a `viewBox` of `0 0 60 60` and a width & height of `60` as well. Should look something like this when you're done:

```
<svg width="60" height="60" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60">
<!-- stuff -->
</svg>
```

Reply:
SVG's are new to me.
When I ran it through the site you sent, I didn't get the same format as in the other samples.
Not sure what sort of settings you need to use on SVG OMG - I used default settings as well as checking almost every option.

Also, the 60 x 60 cuts off part of the image.

--> What do I do in this case? Go back to design and ask them to recreate the image?

Reply:
SVGs are definitely not intuitive. :)

Yeah probably easiest to go back to design and see if they can adjust the image to be 60 x 60. Otherwise you'd need to clip it and scale it. Once you have a 60 x 60 SVG I can help massage it into the right format.